arangod/Agency/Agent.h

```diff
@@ -150,6 +154,18 @@ class Agent final : public arangodb::Thread, public AgentInterface {
 
  private:
 
+  void logsForTrigger();
+
+  /// @brief clear expired polls registered by Agent::poll
+  ///        if qu is nullptr, we're resigning.
+  ///        Caller must have _promLock!
+  void triggerPollsNoLock(
+    query_t qu = nullptr,
+    SteadyTimePoint const& tp = std::chrono::steady_clock::now() + std::chrono::seconds(60));
```

Comment:
What is the reason behind this default value? Do we need default values here?

Reply:
it's for resignation case